cmd/subcommands/staking.go

```diff
@@ -314,32 +314,48 @@ Edit an existing validator"
 				return err
 			}
 
-			commisionRate, err := numeric.NewDecFromStr(commisionRateStr)
-			if err != nil {
-				return err
+			if (len(commisionRateStr) == 0) {
+				commisionRateStr = "0";
```

Comment:
you cannot set it to zero, as the backend will through error as rate is changed more than the maxChangeRate. Any other way to skip updating this field?

Reply:
@coolcottontail

Reply:
I can't think out any other default value for rate. This field must have a value to fill unless the edit-validator command is changed to include some flag.   so I need to mark this field as a required field, same as validator-address ? 

Also check with @rlan35  on this ?